index.html

```diff
@@ -30,5 +30,199 @@ <h1 class="header header-1" id="job">I'm a software developer</h1>
                 </ul>
             </address>
         </header>
+        <main class="container" id="main-container">
+            <div class="header-container" id="header-2-container">
+                <h2 class="header header-2" id="recent">My Recent Works</h2>
+            </div>
+            <article class="container" id="samples-container">
+                <div class="sample-containers" id="sample-1-container">
+                    <div class="sample-img-container" id="sample-1-img-container">
+                    </div>
+                    <div class="sample-img-txt-head-container" id="sample-img-1-text-head-container">
+                        <h3 class="sample-img-head-txt" id="sample-img-1-head-txt">Multi-Post Stories Gain+Glory</h3>
+                    </div>
+                    <div class="code-lang-container" id="code-lang-container-1">
+                        <div class="ruby-on-rails">Ruby on rails</div>
+                        <div class="css">css</div>
+                        <div class="js">Javascript</div>
+                        <div class="html">html</div>
+                    </div>
+                    <div class="btn-container">
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+                <div class="sample-containers" id="sample-2-container">
```

Comment:
- [ ] The work section has the projects shifted to the left side. We want our design to closely follow the provided [Figma template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) as close as possible. 

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/125169592-d3d78300-e1a2-11eb-9f54-42091d783378.png)



**The original design:** 
![image](https://user-images.githubusercontent.com/12745474/125169437-0d5bbe80-e1a2-11eb-9991-6fa4c060fbeb.png)

Let's improve the UI of our design by ensuring elements are **proportionally aligned**, use **font sizes** and **font-weight**, the **exact background  and font colors**, **margins**, and **padding**s matching the provided [Figma template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design:

Reply:
- [ ] The "about me" section is not well aligned with the rest of the other components of the page. Please kindly fix that

Reply:
Not well resolved